gradoop-data-integration/src/main/java/org/gradoop/dataintegration/importer/csv/RowToVertexMapper.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.dataintegration.importer.csv;
+
+import java.util.ArrayList;
+
+import org.apache.flink.api.common.functions.MapFunction;
+import org.gradoop.common.model.impl.properties.Properties;
+import org.gradoop.common.model.impl.properties.PropertyValue;
+
+/**
+ * Map the values of a token separated row of a file to the properties of a EPGM vertex.
+ * Each property value will be mapped to property name the user set.
+ * This names are stored in a list.
+ */
+public class RowToVertexMapper implements MapFunction<String, Properties> {
+
+  /**
+   * Token separator for the csv file.
+   */
+  private String tokenSperator;
+
+  /**
+   * The path to the csv file
+   */
+  private String filePath;
+
+  /**
+   * The name of the properties
+   */
+  private ArrayList<String> propertyNames;
+
+  /**
+   * Create a new RowToVertexMapper
+   * @param filePath the csv file is stored
+   * @param tokenDelimiter in the file is used
+   * @param propertyNames list of the property names
+   */
+  public RowToVertexMapper(String filePath, String tokenDelimiter,
+          ArrayList<String> propertyNames) {
+    this.filePath = filePath;
+    this.tokenSperator = tokenDelimiter;
+    this.propertyNames = propertyNames;
+  }
+
+  @Override
+  public Properties map(final String line) throws Exception {
+
+    return parseProperties(line, propertyNames);
+  }
+
+  /**
+   * Map each label to the occurring properties.
+   * @param line one row of the csv file, contains all the property values of one vertex
+   * @param propertyNames identifier of the property values
+   * @return the properties of the vertex
+   */
+  private Properties parseProperties(String line, ArrayList<String> propertyNames) {
+
+    Properties properties = new Properties();
+
+    String[] propertyValues = line.split(tokenSperator);
+
+    /*
+     * If the line to read is equals to the header, we do not import this line
```

Comment:
Good idea, but the check is very expensive. This check should be optional and the user should indicate whether to check for reoccurring header lines or not.